packages/nx/src/command-line/affected.ts

```diff
@@ -117,16 +117,31 @@ export async function affected(
 
       case 'affected': {
         const projectsWithTarget = allProjectsWithTarget(projects, nxArgs);
-        await runCommand(
-          projectsWithTarget,
-          projectGraph,
-          { nxJson },
-          nxArgs,
-          overrides,
-          null,
-          extraTargetDependencies,
-          { excludeTaskDependencies: false, loadDotEnvFiles: true }
-        );
+        if (nxArgs.graph) {
+          const projectNames = projectsWithTarget.map((t) => t.name);
+
+          return await generateGraph(
+            {
+              watch: true,
+              open: true,
+              view: 'tasks',
+              target: nxArgs.targets[0],
```

Comment:
This means that the generated graph is inaccurate for `nx affected -t build, test` right? The `test` targets wouldn't show up.

Reply:
Support for the multiple targets isn't possible at the moment yet and not in scope for this change but I will pass in the full target array and get the first value in `generateGraph`